package.json

```diff
@@ -43,7 +43,7 @@
     "example:cli2": "./node_modules/.bin/ngc && cp -rf dist example/cli2/node_modules/angular-tree-component/ && cd example/cli2 && ./node_modules/.bin/ng serve",
     "example:cli": "./node_modules/.bin/ngc && cp -rf dist example/cli/node_modules/angular-tree-component/ && cd example/cli && ./node_modules/.bin/ng serve",
     "selenium": "webdriver-manager update && webdriver-manager start",
-    "example:cli:build": "cd example/cli && cp src/index.build.html src/index.html && ./node_modules/.bin/ng build --aot && cp src/index.dev.html src/index.html",
+    "example:cli:build": "cd example/cli && cp src/index.build.html src/index.html && npm install ../.. && ./node_modules/.bin/ng build --aot && cp src/index.dev.html src/index.html",
```

Comment:
Oh this is awesome. So let's use it in `example:cli` task as well instead of `cp -rf`!
Maybe it's better to run `npm run build` in the main directory first (so that `dist` is updated)?